types/serverless/index.d.ts

```diff
@@ -5,7 +5,7 @@
 //                 Alex Pavlenko <https://github.com/a-pavlenko>
 //                 Frédéric Barthelet <https://github.com/fredericbarthelet>
 //                 Bryan Hunter <https://github.com/bryan-hunter>
-//                 Thomas Aribart <https://github.com/thomasaribart>
```

Comment:
You're dropping this GH handle, why?

Reply:
Because apparently I can't copy and paste 🤦

Reply:
Have restored this GH handle :)